icons/b4x.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" xml:space="preserve" width="24" height="24" viewBox="0 0 24 24"><title id="title4">B4X</title><path d="M16.147 0q-.01 0-.022.015c-.207.217-4.936 8.733-4.894 8.812.029.054.182.26.34.457q.372.463.304.414c3.693 3.8 7.446 4.904 11.215 5.909-2.154-1.03-4.3-2.008-6.014-4.884v-.004l-.001-.006.08-.247a.01.01 0 0 1 .007-.008c-.087-.073-.367-.644-.584-1.198-.945-2.404-1.114-5.262-.506-8.55.094-.51.116-.715.075-.71M7.688.49C7.463.49-.102 14.747.001 14.979c2.376 5.364 8.09 8.78 15.043 8.997.224.006.514.016.646.021 1.022.04 3.979-.225 4.555-.393-2.524-.057-5.428-.613-7.772-2.067-2.37-1.472-4.165-3.864-4.408-7.584l.001-.002.001-.005 2.892-5.222.002-.002C9.353 6.859 8.115 3.905 7.826 1.206 7.759.59 7.74.489 7.688.489m14.137 1.658c-.063-.012-.625 1.028-2.366 4.229-1.217 2.236-2.235 4.06-2.282 4.088a.01.01 0 0 1 0 .01l-.078.24c1.741 2.92 3.926 3.877 6.117 4.926l.005.002a.012.012 0 0 1 .009.015.012.012 0 0 1-.015.008c-4.115-1.094-8.22-2.273-12.244-6.915L8.09 13.954c.243 3.711 2.033 6.094 4.397 7.561 2.357 1.464 5.287 2.017 7.822 2.066l.026-.014c.214-.166 3.474-7.751 3.361-7.82-2.945-1.812-3.87-7.533-2.081-12.875.141-.424.245-.719.211-.725" style="display:inline;opacity:1;fill:#000;stroke-width:.0239003;stroke-dasharray:none"/></svg>
```

Comment:
Please see https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#5-check-the-icon, the SVG must not have any other attributes than those listed. That means the `style` attribute on the `path` element, the `id` attribute on the `title` element, the `xml:space`, `width`, and `height` attributes on the root `svg` element.

Reply:
👌